content/zh/docs/reference/kubernetes-api/authorization-resources/cluster-role-binding-v1.md

```diff
@@ -0,0 +1,651 @@
+  - **subjects.namespace** (string)
+    Namespace of the referenced object.  If the object kind is non-namespace, such as "User" or "Group", and this value is not empty the Authorizer should report an error.
+  -->
+  - **subjects.apiGroup** (string)
+    
+    APIGroup 包含被引用主题的 API 组。对于 ServiceAccount 主题默认为 ""。
+    对于 User 和 Group 主题，默认为 "rbac.authorization.k8s.io"。
+  
+  - **subjects.namespace** (string)
+    
+    被引用对象的命名空间。
+    如果对象种类是 "User" 或 "Group" 等非命名空间且该值不为空，
+    则 Authorizer 应报告一个错误。
+<!-- 
+## ClusterRoleBindingList {#ClusterRoleBindingList}
+ClusterRoleBindingList is a collection of ClusterRoleBindings
+
+<hr>
+
+- **apiVersion**: rbac.authorization.k8s.io/v1
+- **kind**: ClusterRoleBindingList
+-->
+## ClusterRoleBindingList {#ClusterRoleBindingList}
+
+ClusterRoleBindingList 是 ClusterRoleBindings 的集合。
+
+<hr>
+- **apiVersion**: rbac.authorization.k8s.io/v1
+
+- **kind**: ClusterRoleBindingList
+<!--
+- **metadata** (<a href="{{< ref "../common-definitions/list-meta#ListMeta" >}}">ListMeta</a>)
+
+  Standard object's metadata.
+
+- **items** ([]<a href="{{< ref "../authorization-resources/cluster-role-binding-v1#ClusterRoleBinding" >}}">ClusterRoleBinding</a>), required
+
+  Items is a list of ClusterRoleBindings
+-->
+- **metadata** (<a href="{{< ref "../common-definitions/list-meta#ListMeta" >}}">ListMeta</a>)
+  
+  标准对象的元数据。
+
+- **items** ([]<a href="{{< ref "../authorization-resources/cluster-role-binding-v1#ClusterRoleBinding" >}}">ClusterRoleBinding</a>)，必选
+  
+  Items 是 ClusterRoleBindings 的列表。
+<!-- 
+## Operations {#Operations}
+
+<hr>
+-->
+## 操作 {#Operations}
+
+<hr>
+<!-- 
+### `get` read the specified ClusterRoleBinding
+#### HTTP Request
+GET /apis/rbac.authorization.k8s.io/v1/clusterrolebindings/{name}
+-->
+
+### `get` 读取指定的 ClusterRoleBinding
+
+#### HTTP 请求
+
+GET /apis/rbac.authorization.k8s.io/v1/clusterrolebindings/{name}
+<!-- 
+#### Parameters
+
+- **name** (*in path*): string, required
+  name of the ClusterRoleBinding
+- **pretty** (*in query*): string
+  <a href="{{< ref "../common-parameters/common-parameters#pretty" >}}">pretty</a>
+-->
+#### 参数
+
+- **name** (*in path*): string，必选
+  
+  ClusterRoleBinding 的名称
+
+- **pretty** (*in query*): string
+  
+  <a href="{{< ref "../common-parameters/common-parameters#pretty" >}}">pretty</a>
+<!-- 
+#### Response
+200 (<a href="{{< ref "../authorization-resources/cluster-role-binding-v1#ClusterRoleBinding" >}}">ClusterRoleBinding</a>): OK
+
+401: Unauthorized
+-->
+#### 响应
+
+200 (<a href="{{< ref "../authorization-resources/cluster-role-binding-v1#ClusterRoleBinding" >}}">ClusterRoleBinding</a>): OK
+
+401: Unauthorized
+<!-- 
+### `list` list or watch objects of kind ClusterRoleBinding
+
+#### HTTP Request
+
+GET /apis/rbac.authorization.k8s.io/v1/clusterrolebindings
+
+#### Parameters
+-->
```

Comment:
I think `#### Parameters` is better to be grouped with next paragraph or standalone.